File: 2MONTH_HW2-master/app/src/main/java/com/example/a2month_hw2/MainActivity.java
package com.example.a2month_hw2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportFragmentManager().beginTransaction().add(R.id.icontainer, new FirstFragment()).commit();
    }
}

File: 2MONTH_HW6-master/app/src/main/java/com/example/hw61/MainActivity.kt
package com.example.hw61

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        supportFragmentManager.beginTransaction().add(R.id.image_fragment, ImageFragment()).commit()

        supportFragmentManager.beginTransaction().add(R.id.songs_fragment, SongsFragment()).commit()
    }
}
